src/installer/pkg/projects/netcoreapp/pkg/Microsoft.NETCore.App.Runtime.pkgproj

```diff
@@ -4,7 +4,7 @@
     Include HostPolicy and HostResolver based on legacy packages. This can be collapsed once the
     legacy packages are no longer used.
   -->
-  <Import Project="..\..\Microsoft.NETCore.DotNetHostPolicy\Microsoft.NETCore.DotNetHostPolicy.props" />
-  <Import Project="..\..\Microsoft.NETCore.DotNetHostResolver\Microsoft.NETCore.DotNetHostResolver.props" />
+  <Import Condition="'$(TargetOS)' != 'iOS'" Project="..\..\Microsoft.NETCore.DotNetHostPolicy\Microsoft.NETCore.DotNetHostPolicy.props" />
+  <Import Condition="'$(TargetOS)' != 'iOS'" Project="..\..\Microsoft.NETCore.DotNetHostResolver\Microsoft.NETCore.DotNetHostResolver.props" />
```

Comment:
Is iOS really what matters here? I would expect Mono (*if* we don't want to include the host resolver or host policy libraries with it).

Reply:
I made it iOS here because mono should also be able to generate the same things as coreclr.  Generating just runtime packs are only going to apply to iOS and Android.

Reply:
I think it'd be nice to make this clear by adding a property and clarifying comment (that doesn't interfere with existing comment):

```xml
<!--
  Don't include host resolver or host policy in iOS or Android runtime packs, because there is no
  shared framework on those platforms.
-->
<PropertyGroup>
  <ExcludeHostAssets Condition="'$(TargetOS)' == 'iOS'">true</ExcludeHostAssets>
  <ExcludeHostAssets Condition="'$(TargetOS)' == 'Android'">true</ExcludeHostAssets>
</PropertyGroup>

<!--
  Include HostPolicy and HostResolver based on legacy packages. This can be collapsed once the
  legacy packages are no longer used.
-->
<Import Condition="'$(ExcludeHostAssets)' != 'true'" Project="..\..\Microsoft.NETCore.DotNetHostPolicy\Microsoft.NETCore.DotNetHostPolicy.props" />
<Import Condition="'$(ExcludeHostAssets)' != 'true'" Project="..\..\Microsoft.NETCore.DotNetHostResolver\Microsoft.NETCore.DotNetHostResolver.props" />
```